transformer_lens/HookedTransformer.py

```diff
@@ -488,12 +493,14 @@ def to_str_tokens(
         elif isinstance(input, torch.Tensor):
             tokens = input
             tokens = tokens.squeeze()  # Get rid of a trivial batch dimension
+            tokens = tokens.unsqueeze(0) if tokens.dim() == 0 else tokens  # Don't pass dimensionless tensor
```

Comment:
Can you make this into a proper if statement? Reading the code it's a bit confusing to see `tokens.squeeze()` and then on the next line `tokens.unsqeeze(0)` before I noticed the inline if